bin/server/cmd/root.go

```diff
@@ -124,6 +127,7 @@ func init() {
 	rootCmd.PersistentFlags().BoolVar(&readonly, "readonly", false, "whether to run in read-only mode")
 	rootCmd.PersistentFlags().BoolVar(&demo, "demo", false, "whether to run using demo data")
 	rootCmd.PersistentFlags().BoolVar(&debug, "debug", false, "whether to enable debug level logging")
+	rootCmd.PersistentFlags().StringVar(&pgURL, "pg", "", "optional external postgresql instance url")
```

Comment:
Let's add a comment for an example url string and also explain a bit background of the chosen url scheme.

Reply:
Do you mean a comment in code or the cmd output?

Reply:
Why not both :)

Reply:
BTW, one of the main reasons I'd like to see the format is because I can't think of a de facto URL format, so I am interested:)